app/components/AddThis.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+interface Props {
+  url: string;
+  title: string;
+}
+
+declare const window: any;
+
+export default class Card extends React.Component<Props, {}> {
+  componentDidMount() {
+    window.addEventListener('load', window.addthis.layers.refresh());
+  }
+
+  componentDidUpdate() {
+    window.addEventListener('load', window.addthis.layers.refresh());
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('load', window.addthis.layers.refresh());
```

Comment:
Forgot to mention here: This doesn't do anything.

window can't remove a function being called. 

If you wan't to remove the `load` listener associated with `refresh`, you need to write it without the `()`.

Also, is it `window.addthis` or `window.addThis`???

Reply:
cc: @jvoros

Reply:
Good catch on the function invocation.

lowercase t.